testworld/park.go

```diff
@@ -0,0 +1,248 @@
+// +build testworld
+
+package testworld
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+
+	"testing"
+
+	"time"
+
+	"github.com/centrifuge/go-centrifuge/bootstrap"
+	"github.com/centrifuge/go-centrifuge/cmd"
+	"github.com/centrifuge/go-centrifuge/config"
+	ctx "github.com/centrifuge/go-centrifuge/context"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/centrifuge/go-centrifuge/node"
+	"github.com/gavv/httpexpect"
+	logging "github.com/ipfs/go-log"
+)
+
+var log = logging.Logger("host")
+
+var hostconfig = []struct {
+	name             string
+	apiPort, p2pPort int64
+}{
+	{"Alice", 8084, 38204},
+	{"Bob", 8085, 38205},
+	{"Charlie", 8086, 38206},
+}
+
+// manager is the manager of the hosts at Testworld (Robert)
+type manager struct {
+
+	// network settings
+	ethNodeUrl, accountKeyPath, accountPassword, network string
+
+	txPoolAccess bool
+
+	// contractAddresses are the addresses of centrifuge contracts on Ethereum
+	contractAddresses *config.SmartContractAddresses
+
+	// bernard is the bootnode for all the hosts
+	bernard *host
```

Comment:
Why not call it `mainBootNode` or similar?

Reply:
this follows `Westworld` analogy ;)